projects/lwan/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -eu
+
+mkdir -p $WORK/lwan
+cd $WORK/lwan
+
+cmake -GNinja \
+	-DCMAKE_BUILD_TYPE=Debug -DSANITIZER=${SANITIZER} \
```

Comment:
How important the `$SANITIZER` value is? We generally don't recommend to rely on env variables other than the ones specified here: https://github.com/google/oss-fuzz/blob/master/docs/new_project_guide.md#requirements

Reply:
Indeed, with `-DCMAKE_C_FLAGS`, this is not needed.